elasticsearch_dsl/response/__init__.py

```diff
@@ -123,33 +153,42 @@ def search_after(self):
         """
         if len(self.hits) == 0:
             raise ValueError("Cannot use search_after when there are no search results")
-        if not hasattr(self.hits[-1].meta, "sort"):
+        if not hasattr(self.hits[-1].meta, "sort"):  # type: ignore
```

Comment:
This and the next one below will probably start working once I type more of the response object.